### PR TITLE
react-toggle-switch-component | Wonseop Lee

### PR DESCRIPTION
**Pull request by wonseop-lee**
react-toggle-switch-component

### CODE REVIEW

File: react-toggle-switch-component/src/toggle-switch.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+class ToggleSwitch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```
Comment (BrettAlbright) [inline react-toggle-switch-component/src/toggle-switch.jsx:6]: While this does work, I would remove everything from state except the `isToggled` property. The reason for this is that you always want to store in state only what is necessary for the ui to be properly updated. You can set the appropriate classnames and such within the handleToggle function.

File: react-toggle-switch-component/src/toggle-switch.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      onOrOff: 'OFF'
+    };
+    this.handleToggle = this.handleToggle.bind(this);
+  }
+
+  handleToggle() {
+    const isToggled = this.state.isToggled;
+    if (isToggled) {
+      this.setState({ isToggled: false });
+      this.setState({ className: 'notToggled' });
```
Comment (BrettAlbright) [inline react-toggle-switch-component/src/toggle-switch.jsx]: for future reference, setState takes an object as an argument, so if you ever have to update multiple properties at once, you can just pass it an object with each of the properties you want to update.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small updates

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**